path_config.go

```diff
@@ -14,10 +14,18 @@ import (
 	"github.com/hashicorp/go-cleanhttp"
 	"github.com/hashicorp/vault/sdk/framework"
 	"github.com/hashicorp/vault/sdk/helper/certutil"
+	"github.com/hashicorp/vault/sdk/helper/strutil"
```

Comment:
These don't parse for me at first until I run `$ go mod vendor`, and then I get a diff in vendored files. So I think we need `$ go mod vendor` run on this PR.

Reply:
This is addressed in https://github.com/hashicorp/vault-plugin-auth-jwt/pull/99 if we merge it.